drivers/sensor/tdk/icm4268x/icm4268x_decoder.c

```diff
@@ -735,13 +751,13 @@ static bool icm24688_decoder_has_trigger(const uint8_t *buffer, enum sensor_trig
 }
 
 SENSOR_DECODER_API_DT_DEFINE() = {
-	.get_frame_count = icm42688_decoder_get_frame_count,
-	.get_size_info = icm42688_decoder_get_size_info,
-	.decode = icm42688_decoder_decode,
+	.get_frame_count = icm4268x_decoder_get_frame_count,
+	.get_size_info = icm4268x_decoder_get_size_info,
+	.decode = icm4268x_decoder_decode,
 	.has_trigger = icm24688_decoder_has_trigger,
```

Comment:
would be nice to get that typo fixed

Reply:
Thanks for spotting this. Addressed in #93742